services/m3aggregator/config/aggregator.go

```diff
@@ -46,6 +46,10 @@ import (
 	"github.com/spaolacci/murmur3"
 )
 
+const (
+	initialBufferSizeGrowthFactor = 2
```

Comment:
It doesn't have to be 1.5 times, that was just for illustration. I'm just thinking that if the buffer is, for example, 10 kB, it might make more sense to allocate a buffer that is 12 kB instead of 20 kB, since the the overflow will likely be less than 2kB.